src/qt/bitcoin.cpp

```diff
@@ -442,7 +442,7 @@ void BitcoinApplication::requestShutdown()
 
 #ifdef ENABLE_WALLET
     window->removeAllWallets();
-    for (WalletModel *walletModel : m_wallet_models) {
+    for (const WalletModel *walletModel : m_wallet_models) {
```

Comment:
Whitespace

Reply:
I'm not sure I understand what whitespace change you're suggesting? :-)

Reply:
Just that the developer-notes prefer `WalletModel*`. Fine to ignore.